tests/coalaCITest.py

```diff
@@ -11,6 +12,8 @@ class coalaCITest(unittest.TestCase):
 
     def setUp(self):
         self.old_argv = sys.argv
+        self.unescaped_coafile = os.path.abspath('./.coafile')
+        self.coafile = PathArg(self.unescaped_coafile)
```

Comment:
These 2 lines are obsolete and were already removed by @clarammdantas via https://github.com/coala/coala/pull/5012